docs/getting-started/smr-disk.md

```diff
@@ -11,33 +11,35 @@ technology can have different interface implementations. This results in
 different usage models:
 
 * **Drive Managed Interface:** SMR disks that implement this interface are seen
-  by the host kernel and applications as identical to regular disks and do not
-  need special attention. Most Linux&reg; kernels recognize these disks. With
-  this implementation, no LBA space zoning information is available to the
-  host. Drive Managed disks are therefore not considered to be zoned block
-  devices.
-
-* **Zoned Block Interface:** SMR hard-disk drives that implement ZBC and ZAC
-  feature sets provide commands to the host that allow the host to indentify and
-  control the device zones. This interface has two different variations, or
-  model implementations (See [SMR Interface Implementations](../introduction/smr#smr-implementations)):
-
-	* **Host Aware:** While this zone model offers the convenience and
-	  flexibility of Drive Managed disks (for example, random write
-	  capabilities), Host Aware disks support the full set of zone commands
-	  defined by ZBC and ZAC standards. Host Aware disks can support both
-	  the regular block device abstraction ("regular disk") and the
-	  zoned block device abstraction.
+  by the host kernel and by applications as identical to regular disks. Most
+  Linux&reg; kernels recognize these disks. SMR disks that implement this
+  interface make no LBA-space zoning information available to the host. Drive
+  Managed disks are therefore not considered to be zoned block devices.
+
+* **Zoned Block Interface:** SMR hard-disk drives that implement the ZBC and 
+  ZAC feature sets also provide commands to the host that allow the host to 
+  identify and control the device's zones. This interface has two different 
+  variations, or model implementations (See [SMR Interface Implementations](../introduction/smr#smr-interface-implementations)). Those two model implementations are:
+
+	* **Host Aware:** This zone model offers the convenience and
+	  flexibility of Drive Managed disks (for example, it provides 
+          random write capabilities) and also supports the full set of 
+          zone commands defined in the ZBC and the ZAC standards. Host 
+          Aware disks can support both the regular block device 
+          abstraction ("regular disk") and the zoned block device 
+          abstraction.
```

Comment:
Weird indentation... Mixed up spaces and tabs ?

Reply:
I've just looked at this carefully, and the only thing that I can guess is that you're not keen on vim's auto-indentation (gq) in Visual Mode. So I've manually added carriage returns and spaces in order to make this consistent with itself.

It built with no problems locally both ways.

Reply:
This looks fine in the repository when you examine the file with a text editor, but it renders with the weird spacing here in Github.

This is weird.

Reply:
Using "set syntax=whitespace" in vim, I seem to have corrected the strange indentation. However, I remain uncertain of exactly what changed. The highlighted syntax looked the same before the change as it did after the change.

Despite my continuing confusion in this connection, I think this is repaired.